modules/33acrossBidAdapter.js

```diff
@@ -320,7 +354,19 @@ function buildRequests(bidRequests, bidderRequest) {
 
   adapterState.uniqueSiteIds = bidRequests.map(req => req.params.siteId).filter(utils.uniques);
 
-  return bidRequests.map(req => _createServerRequest(req, gdprConsent, uspConsent, pageUrl));
+  let getFloor;
+  if (typeof bidRequests.getFloor === 'function') {
```

Comment:
interesting, i thought the `getFloor()` is in each of the `bidRequest`

Reply:
You are correct, it's set on the individual bid request object. I will fix this.